presto-main/etc/catalog/pinot.properties

```diff
@@ -0,0 +1,11 @@
+connector.name=pinot
+
+pinot.controller-urls=localhost:9000
+
+# Infer Pinot time fields to Presto Date/Timestamp type
+pinot.infer-date-type-in-schema=true
+pinot.infer-timestamp-type-in-schema=true
+
+# Use Pinot SQL endpoint for broker queries with aggregation pushdown.
+# Pinot version >= 0.4.0 is required.
+pinot.use-pinot-sql-for-broker-queries=true
```

Comment:
Since we are anyway filing a prestodb diff here: How about we make these default set in the code itself and thin this config file even more ? I think the only reason not to do that would be that these settings are only supported on a newer Pinot cluster. But that is also true of this 'sample config' file. 

If you do end up moving these configs there, we can also add this comment `Pinot version >= 0.4.0 is required` to the `PinotConfig.java` file.

Reply:
done